detectron2/utils/patch_instance.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import importlib.util
+import os
+import sys
+import tempfile
+import torch
+
+from detectron2.structures import Boxes, Instances  # noqa F401
+
+
+def export_torchscript_with_patch_instance(model, fields):
+    with tempfile.NamedTemporaryFile(mode="w", encoding="utf-8", suffix=".py", delete=False) as f:
+        cls_name, s = gen_class(fields)
+        f.write(s)
+        f.flush()
+        f.close()
+
+        module = _import(f.name)
+        new_instance = getattr(module, cls_name)
+        _ = torch.jit.script(new_instance)
+
+        Instances.__torch_script_class__ = True
+        Instances._jit_override_qualname = torch._jit_internal._qualified_name(new_instance)
+        return torch.jit.script(model), new_instance
+
+
+def _typing_imports_str():
```

Comment:
return a multi-line string directly to be more readable

Reply:
done